docs/extend/extend-apiml/api-mediation-security.md

```diff
@@ -641,17 +642,28 @@ in following shell scripts:
 
 ## Security Service Client library
 
-The `security-service-client-spring` library contains authentication providers and filters to handle the authentication against the z/OSMF service. 
-There are two providers:
+The `security-service-client-spring` library provides authentication and protection against security providers.
```

Comment:
I'm not sure "against" is the right word here. Maybe "for" is more suitable? Let me talk to you in person to determine how to formulate this one.